auto_reconcile_structured_ref/models/account_bank_statement_line.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api
+from odoo.exceptions import UserError
+
+
+class AccountBankStatementLine(models.Model):
+    _inherit = "account.bank.statement.line"
+
+    @api.multi
+    def auto_reconcile(self):
+        # If we don't have esr_reconcile we want the default odoo way
+        if not self.env.context.get('struct_reconcile'):
```

Comment:
struct_reconcile -> specific_reconcile
so if there is a need only method with initializing records need to replaced